jcl/src/java.base/share/classes/java/lang/invoke/VarHandleGuards.java

```diff
@@ -1,7 +1,7 @@
-/*[INCLUDE-IF Sidecar19-SE-B174]*/
+/*[INCLUDE-IF Sidecar19-SE]*/
```

Comment:
Shouldn't this be `Sidecar19-SE-OpenJ9`?  Here and below

Reply:
Technically yes, but using Sidecar19-SE doesn't break anything. I expect Sidecar19-SE-OpenJ9 to be replaced with Sidecar19-SE once SIDECAR19-SE-B148 is deleted, so using Sidecar19-SE now saves some changes later.